IdentityCore/src/parameters/MSIDRequestParameters.h

```diff
@@ -60,6 +60,8 @@
 @property (nonatomic) NSString *intuneApplicationIdentifier;
 @property (nonatomic) MSIDRequestType requestType;
 @property (nonatomic) MSIDCurrentRequestTelemetry *currentRequestTelemetry;
+// indicates if current process is SSO extension
+@property (nonatomic,readonly) BOOL isFromSSOExtension;
```

Comment:
Added a comment in broker repo as well, but I think parameters is not the best place for this flag. Better to have it in constructor of the controller or not have it at all.